[2022-05-30 20:21:32,433] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [queued]>
[2022-05-30 20:21:32,912] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [queued]>
[2022-05-30 20:21:32,914] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 20:21:32,927] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 20:21:32,928] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 20:21:33,409] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-09-02 06:00:00+00:00
[2022-05-30 20:21:33,965] {standard_task_runner.py:52} INFO - Started process 1301 to run task
[2022-05-30 20:21:34,139] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-09-02T06:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmp60rbxkty', '--error-file', '/tmp/tmprl2_kmjg']
[2022-05-30 20:21:34,207] {standard_task_runner.py:80} INFO - Job 140: Subtask ingest
[2022-05-30 20:21:37,069] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [running]> on host e083cbf9476b
[2022-05-30 20:21:37,431] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 20:21:37,711] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-09-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-02T06:00:00+00:00
[2022-05-30 20:21:37,717] {logging_mixin.py:115} INFO - yellow_taxi_2021_09 /opt/***/output_2021-09.parquet 2021-09-02T06:00:00+00:00
[2022-05-30 20:21:37,741] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ingest_script.py", line 12, in ingest_callable
    engine = create_engine(f'postgresql://{user}:{password}@{host}:{port}/{db}')
  File "<string>", line 2, in create_engine
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/deprecations.py", line 298, in warned
    return fn(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/create.py", line 518, in create_engine
    u = _url.make_url(url)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/url.py", line 711, in make_url
    return _parse_rfc1738_args(name_or_url)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/url.py", line 772, in _parse_rfc1738_args
    components["port"] = int(components["port"])
ValueError: invalid literal for int() with base 10: 'None'
[2022-05-30 20:21:37,899] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20210902T060000, start_date=20220530T202132, end_date=20220530T202137
[2022-05-30 20:21:38,140] {standard_task_runner.py:97} ERROR - Failed to execute job 140 for task ingest (invalid literal for int() with base 10: 'None'; 1301)
[2022-05-30 20:21:41,208] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-05-30 20:21:41,463] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-30 22:16:58,128] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [queued]>
[2022-05-30 22:16:58,169] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [queued]>
[2022-05-30 22:16:58,172] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:16:58,183] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 22:16:58,198] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:16:58,255] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-09-02 06:00:00+00:00
[2022-05-30 22:16:58,375] {standard_task_runner.py:52} INFO - Started process 1371 to run task
[2022-05-30 22:16:58,401] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-09-02T06:00:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpc65r77if', '--error-file', '/tmp/tmph7c90ute']
[2022-05-30 22:16:58,410] {standard_task_runner.py:80} INFO - Job 219: Subtask ingest
[2022-05-30 22:16:58,734] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [running]> on host 8ac972de2f4f
[2022-05-30 22:16:58,965] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 22:16:59,497] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-09-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-02T06:00:00+00:00
[2022-05-30 22:16:59,603] {logging_mixin.py:115} INFO - yellow_taxi_2021_09 /opt/***/output_2021-09.parquet 2021-09-02T06:00:00+00:00
[2022-05-30 22:17:00,042] {logging_mixin.py:115} INFO - connection established successfully, inserting data...
[2022-05-30 22:17:35,275] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2022-05-30 22:17:35,362] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20210902T060000, start_date=20220530T221658, end_date=20220530T221735
[2022-05-30 22:17:35,755] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-30 22:21:03,725] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [queued]>
[2022-05-30 22:21:03,756] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [queued]>
[2022-05-30 22:21:03,770] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:21:03,773] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-30 22:21:03,776] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 22:21:03,843] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): ingest> on 2021-09-02 06:00:00+00:00
[2022-05-30 22:21:03,880] {standard_task_runner.py:52} INFO - Started process 1725 to run task
[2022-05-30 22:21:03,907] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-09-02T06:00:00+00:00', '--job-id', '248', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmp6lpjz1lg', '--error-file', '/tmp/tmpy46icaff']
[2022-05-30 22:21:03,914] {standard_task_runner.py:80} INFO - Job 248: Subtask ingest
[2022-05-30 22:21:04,082] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-09-02T06:00:00+00:00 [running]> on host 8ac972de2f4f
[2022-05-30 22:21:04,175] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-30 22:21:04,255] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-09-02T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-02T06:00:00+00:00
[2022-05-30 22:21:04,264] {logging_mixin.py:115} INFO - yellow_taxi_2021_09 /opt/***/output_2021-09.parquet 2021-09-02T06:00:00+00:00
[2022-05-30 22:21:04,309] {logging_mixin.py:115} INFO - connection established successfully, inserting data...
[2022-05-30 22:21:48,169] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2022-05-30 22:21:48,746] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20210902T060000, start_date=20220530T222103, end_date=20220530T222148
[2022-05-30 22:21:49,864] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
